client/pipeline_test.go

```diff
@@ -26,7 +26,7 @@ const testFbitConfigWithAddr3 = `[INPUT]
 	Port     5140
 	Mode     tcp
 [INPUT]
-	Name     syslog.1
```

Comment:
@pwhelan can you confirm me that this was invalid?

Reply:
that's invalid.

Reply:
Thanks @edsiper 
We had some tests in Cloud back-end that were not too strict. I change it to be stricter and cause this one to fail.